fluentbit/td-agent-bit.repo

```diff
@@ -0,0 +1,6 @@
+[td-agent-bit]
```

Comment:
@EvgeniyPatlan is it ok to enable additional external repo?

Reply:
I removed it from default build flow, but left possibility to use it with `--build-arg FLUENTBIT_SOURCE=fluentbit_repository`